src/Service.Tests/dab-config.CosmosDb_NoSql.json

```diff
@@ -34,7 +34,7 @@
     }
   },
   "entities": {
-    "Planet": {
+    "PlanetAlias": {
```

Comment:
in tests it seems like you're removing "Alias" from the entity names, but here you're adding it? I'm not following the naming changes here.

Reply:
Yes because Alias is only applicable on "Model" and for Cosmos DB only container level entity can be a model. Hence, putting alias for other entities, can be confusing in test, for Cosmos DB perspective. 
Hence, I modified the tests and config to reflect the same intention.